cfgov/ask_cfpb/views.py

```diff
@@ -131,3 +134,38 @@ def ask_autocomplete(request, language='en'):
                 'url': result.url}
                for result in sqs[:20]]
     return JsonResponse(results, safe=False)
+
+
+def redirect_ask_search(request, language='en'):
+    """
+    Catch old English pages built via query strings and
+    redirect them to category pages if we can. If the query string
+    has a 'q' query, we'll run that search.
+    """
+
+    cat_string = 'category_exact:'  # the facet prefix for categories
+    if request.GET.get('q'):
+        querystring = request.GET.get('q')
+        if not querystring:
+            raise Http404
+        return redirect(
+            '/ask-cfpb/search/?q={query}'.format(
+                query=querystring, permanent=True))
+    else:
+        facets = request.GET.getlist('selected_facets')
+        if not facets:
+            raise Http404
+        # Get the category, if there is one
+        for facet in facets:
+            if cat_string in facet:
+                category = facet.replace(cat_string, '')
+        if not category:
+            raise Http404
+        if language == 'en':
```

Comment:
Since you take a default `language='en'`, would you want to make this condition the default return value too, so it always returns?

```python
if language == 'es':
    return redirect('/es/obtener-respuestas/categoria-{category}'…)

return redirect('/ask-cfpb/category-{category}'…)
```